features/bevragen/persoon-beperkt/verblijfplaats/dev/fields-gba.feature

```diff
@@ -0,0 +1,394 @@
+    | verblijfplaatsBinnenland.functieAdres.code         |
+    | verblijfplaatsBinnenland.functieAdres.omschrijving |
+
+  Scenario: locatie wordt gevraagd met field pad 'verblijfplaatsBinnenland.type'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                        | waarde                      |
+    | gemeentecode (92.10)        | 0518                        |
+    | locatiebeschrijving (12.10) | Woonboot bij de Grote Sloot |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | verblijfplaatsBinnenland.type       |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                | waarde                      |
+    | locatiebeschrijving | Woonboot bij de Grote Sloot |
+
+Rule: Een persoon heeft een verblijfplaats zonder gegevens/leeg verblijfplaats object wanneer de verblijfplaatsBinnenland field alias wordt gebruikt en de persoon heeft een verblijfplaats buitenland of verblijfplaats onbekend
+
+  Abstract Scenario: de persoon heeft een verblijfplaats buitenland en één of meerdere velden wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                          | waarde |
+    | land adres buitenland (13.10) | 6014   |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon zonder 'verblijfplaats' gegevens
+
+    Voorbeelden:
+    | fields                                |
+    | verblijfplaatsBinnenland              |
+    | verblijfplaatsBinnenland.functieAdres |
+    | verblijfplaatsBinnenland.type         |
+
+Rule: als één of meerdere (gegevensgroep) velden van een verblijfplaats wordt gevraagd en de categorie verblijfplaats (08) heeft in onderzoek gegevens, dan worden deze ook geleverd
+
```

Comment:
Is dit niet exclusief de GBA gegevens die corresponderen met adressering (aanduiding bij huisnummer, postcode, gemeentedeel, gemeente van inschrijving)? Hiervoor moet men dan toch met fields naar adressering vragen? Ik zie waarschijnlijk iets over het hoofd....

Reply:
Het gaat erom dat als in onderzoek gegevens altijd wordt geleverd. Ongeacht of de geleverde velden in onderzoek zijn

Reply:
> Is dit niet exclusief de GBA gegevens die corresponderen met adressering (aanduiding bij huisnummer, postcode, gemeentedeel, gemeente van inschrijving)? Hiervoor moet men dan toch met fields naar adressering vragen? Ik zie waarschijnlijk iets over het hoofd....

Dat is in lijn met eerdere afspraken. Ik heb uitzonderingen moeten toevoegen zodat persoonInOnderzoek en gezagInOnderzoek niet worden geleverd als er geen persoon of gezag categorie rubrieken worden gevraagd in de fields.  Deze rule zegt in principe hetzelfde over verblijfplaats. Adressering gebruikt in essentie verblijfplaats kenmerken dus bij het bevragen van de adressering dan dient verblijfplaats ook te worden opgevraagd. Enig uitzondering zou adressering.Aanhef en dergelijke zijn.